fluent/migrate/context.py

```diff
@@ -131,7 +127,7 @@ def get_sources(acc, cur):
         # may fail but a single missing legacy resource doesn't mean that the
         # migration can't succeed.
         for dependencies in self.dependencies.values():
-            for path in set(path for path, _ in dependencies):
+            for path in {path for path, _ in dependencies}:
```

Comment:
My brain has a hard time parsing this change 🤔

Reply:
My brain has a hard time figuring out what's going on here in the first place without any type hints.

This was a change coming from automation, and I'm relatively confident that the behaviour isn't changed from the previous.